README.md

```diff
@@ -60,7 +65,7 @@ The default choice will include all of the listed features. The features are as
     unwanted secrets/keys committed
   - Pre-push hook - this hook is just a placeholder. For now no action will happen here, but you can configure it by your needs
 
-- Dockerizing GitHub workflow step - this option will create a GitHub workflow file for building a Docker images
+- Containerization with Docker - this option will create a GitHub workflow file for building a Docker images
```

Comment:
This feature will add a `Dockerfile` and a the following scripts: `image:build`, `image:run`